src/Draco.Compiler.Toolset/build/Draco.Compiler.Toolset.targets

```diff
@@ -0,0 +1,24 @@
+<Project>
+    
+  <Target Name="CoreCompile" DependsOnTargets="$(CoreCompileDependsOn)">
+
+    <PropertyGroup Condition="'$(DOTNET_HOST_PATH)' == ''">
+        <DotNetHostDirectory>$(NetCoreRoot)</DotNetHostDirectory>
+        <DotNetHostFileName>dotnet</DotNetHostFileName>
+        <DotNetHostFileName Condition="'$(OS)' == 'Windows_NT'">dotnet.exe</DotNetHostFileName>
+    </PropertyGroup>
+
+    <DracoCompiler ToolPath="$(DotNetHostDirectory)"
+                   ToolExe="$(DotNetHostFileName)"
+                   DracoCompilerPath="$(MSBuildThisFileDirectory)..\tools\net7.0\Draco.Compiler.Cli.dll"
```

Comment:
Should we use `$(DracoToolsTargetFramework)` here? Or rather, can we?

Reply:
We can't (unless we do the same trick as the SDK and replace the version at pack time; I didn't feel like doing it). I think it's probably fine to delete it, as we can't really derive any useful info from this. The app is run through `dotnet`, and we don't know if `dotnet` will find a suitable version of the runtime to use. So, including this is just extra headache.

Conveniently, I already deleted this subfolder from the packing code, but apparently, I forgot to delete it from here. Yeah, this needs to be fixed.